src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/domain/move/MoveQueryRepository.kt

```diff
@@ -222,7 +222,7 @@ class MoveQueryRepository(@Autowired val jdbcTemplate: JdbcTemplate) {
   }
 
   fun movesInDateRange(supplier: Supplier, startDate: LocalDate, endDateInclusive: LocalDate) =
-    MoveType.values().map { movesForMoveTypeInDateRange(supplier, it, startDate, endDateInclusive) }
+    MoveType.values().associateBy({ it }, { movesForMoveTypeInDateRange(supplier, it, startDate, endDateInclusive) })
```

Comment:
Now returns a Map<MoveType, List<Move>() instead of List<List<Move>>.  The calling code was going on the index to determine the move type which was obviously not ideal.